portal/templates/flask_user/emails/eproms_password_changed_message.html

```diff
@@ -3,8 +3,20 @@
 {% block message %}
 <p>Your password has been changed.</p>
 {% if user_manager.enable_forgot_password %}
+
+{% if user.organizations.count() == 1 %}
+<p>If you did not initiate this password change, <a href="{{ url_for('user.forgot_password', _external=True) }}">please click here to reset it</a>, or contact your representative at {{ user.organizations.first().name }}.</p>
+{% else %}
 <p>If you did not initiate this password change, <a href="{{ url_for('user.forgot_password', _external=True) }}">please click here to reset it</a>, or contact your representative at {{ app_name }}.</p>
+{% endif %}
+
+{% else %}
```

Comment:
@pbugni the else is a part of another if statement (this new statement is embedded within that statement)

as for indentation, I just wasn't sure what the standard was for these html templates